packages/compass-home/src/plugin.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import AppRegistry from 'hadron-app-registry';
+
+import Home from './components/home';
+import AppRegistryContext from './contexts/app-registry-context';
+
+import './index.less';
+
+function Plugin({
+  appName,
+  appRegistry,
+}: {
+  appName: string;
+  appRegistry: AppRegistry;
+}): React.ReactElement {
+  return (
+    <AppRegistryContext.Provider value={appRegistry}>
+      <Home appName={appName} />
```

Comment:
We could add a provider for `appName` too, right now it's not as used as app registry so I think probably not doing it yet.